pkg/flexvolume/flexvolume.go

```diff
@@ -93,37 +93,37 @@ func ExitWithResult(result DriverStatus) {
 	res, err := json.Marshal(result)
 	if err != nil {
 		log.Printf("Error marshaling result: %v", err)
-		fmt.Fprintln(out, `{"status":"Failure","message":"Error marshaling result to JSON"}`)
+		fmt.Println(`{"status":"Failure","message":"Error marshaling result to JSON"}`)
 	} else {
 		s := string(res)
 		log.Printf("Command result: %s", s)
-		fmt.Fprintln(out, s)
+		fmt.Println(s)
 	}
-	exit(code)
+	os.Exit(code)
 }
 
 // Fail creates a StatusFailure Result with a given message.
-func Fail(a ...interface{}) DriverStatus {
-	msg := fmt.Sprint(a...)
+func Failf(s string, a ...interface{}) DriverStatus {
+	msg := fmt.Sprintf(s, a...)
```

Comment:
For consistency with other funcs, remove assignment i.e 

Message: fmt.Sprintf(s, a...)